v7/navstories/README.md

```diff
@@ -1,3 +1,34 @@
 This very simple plugin throws all the stories to the navigation bar.
 
-Proof-of-concept of a ConfigPlugin.  By request on the mailing list.
+Started as a Proof-of-concept of a ConfigPlugin.  By request on the mailing list.
+
+Changed to map the stories to a hierachical structure in the menu
```

Comment:
Wait, how deep down does this go? Do you realise you can have only one level of submenus?